doc/source/dev/governance/governance.rst

```diff
@@ -0,0 +1,310 @@
+existing Core Developers who, as of January 2017, have been significantly
+active over the last two years.
+
+When considering potential Members, the Council will look at candidates with a
+comprehensive view of their contributions. This will include but is not limited
+to code, code review, infrastructure work, mailing list and chat participation,
+community help/building, education and outreach, design work, etc. We are
+deliberately not setting arbitrary quantitative metrics (like “100 commits in
+this repo”) to avoid encouraging behavior that plays to the metrics rather than
+the project’s overall well-being. We want to encourage a diverse array of
+backgrounds, viewpoints and talents in our team, which is why we explicitly do
+not define code as the sole metric on which council membership will be
+evaluated.
+
+If a Council member becomes inactive in the project for a period of one year,
+they will be considered for removal from the Council. Before removal, inactive
+Member will be approached by the BDFL to see if they plan on returning to
+active participation. If not they will be removed immediately upon a Council
+vote. If they plan on returning to active participation soon, they will be
+given a grace period of one year. If they don’t return to active participation
+within that time period they will be removed by vote of the Council without
+further grace period. All former Council members can be considered for
+membership again at any time in the future, like any other Project Contributor.
+Retired Council members will be listed on the project website, acknowledging
+the period during which they were active in the Council.
+
+The Council reserves the right to eject current Members, other than the BDFL,
+if they are deemed to be actively harmful to the project’s well-being, and
+attempts at communication and conflict resolution have failed.
+
+A list of current Steering Council Members is maintained at the
+page :ref:`governance-people`.
+
+Conflict of interest
+~~~~~~~~~~~~~~~~~~~~
+
+It is expected that the BDFL and Council Members will be employed at a wide
+range of companies, universities and non-profit organizations. Because of this,
+it is possible that Members will have conflict of interests. Such conflict of
+interests include, but are not limited to:
+
+-   Financial interests, such as investments, employment or contracting work,
+    outside of The Project that may influence their work on The Project.
+-   Access to proprietary information of their employer that could potentially
+    leak into their work with the Project.
+
+All members of the Council, BDFL included, shall disclose to the rest of the
+Council any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Council discussions on that
+issue, but must recuse themselves from voting on the issue. If the BDFL has
+recused his/herself for a particular decision, they will appoint a substitute
```

Comment:
"they" is totally fine, but "his/herself" and "they" in the same line is a bit odd, might as well make it "themself"...